.github/internal/plans/docs/06-error-handling.md

```
@@ -0,0 +1,117 @@
+func ErrorHandler(next http.Handler) http.Handler {
+    return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+        defer func() {
+            if err := recover(); err != nil {
+                httpapi.WriteProblem(w, 500, "Internal server error")
+            }
+        }()
+        next.ServeHTTP(w, r)
+    })
+}
+```
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)